agrirouter-sdk-dotnet-standard-api/Service/Messaging/IMessagingService.cs

```diff
@@ -14,5 +15,12 @@ public interface IMessagingService<in T>
         /// <param name="parameters">Parameters for message sending.</param>
         /// <returns>-</returns>
         MessagingResult Send(T parameters);
+
+        /// <summary>
+        ///     Sending a message using the given message parameters.
```

Comment:
The comment should be adjusted.

```suggestion
        ///     Ansynchronously sending a message using the given message parameters.
```

Reply:
Fixed